Gulpfile.js

```diff
@@ -453,8 +453,7 @@ task("runtests").flags = {
     "-t --tests=<regex>": "Pattern for tests to run.",
     "   --failed": "Runs tests listed in '.failed-tests'.",
     "-r --reporter=<reporter>": "The mocha reporter to use.",
-    "-d --debug": "Runs tests in debug mode (NodeJS 6 and earlier)",
-    "-i --inspect": "Runs tests in inspector mode (NodeJS 8 and later)",
+    "-i --break": "Runs tests in inspector mode (NodeJS 8 and later)",

```

Comment:
Do we really want `-i` to be the shorthand for `--break?

Reply:
🤷 it's backward compatible

Reply:
I'd sorta doubt anyone is taking a hard automated dependency on it, so I wouldn't worry about that too much. 😄

Either way I'll leave it up to you

Reply:
`-b` is already taken by `--browser` despite not being documented in the Gulpfile, so I’m not sure what else to call it.

Reply:
Well...who even runs that one? Anyway, it's not a big deal.